ios/FluentUI/People Picker/Avatar.swift

```diff
@@ -36,6 +40,8 @@ open class AvatarData: NSObject, Avatar {
 
     @objc public var customBorderImage: UIImage?
 
+    @objc public var borderColor: UIColor?
```

Comment:
not sure. I would have imagine the UI is more like the current Office iOS collab menu, which the ring is around the avatar and it could animate around if people is looking at the document and static or gray off if users isn't looking at the document.

Reply:
should the bordercolor just be the same as avatar initial background color logic?

Reply:
The ring animation for presence is not a requirement so let's ignore that for now.
This implementation is similar to what we have for the Office iOS collab menu. To meet the requirements of both Office iOS and Teams, I'll make this change:
Instead of a "border color", I will rename this property to simply "color", representing that this is the color of the avatar. If this color is not specified, by default, the initials color will be used and the white/black border will be used. If the color is set, this color will override the default colors for both the border and the initials. 
This is an important requirement. Having worked a lot in the collab space, we often want to override the color that was generated from the initials. In Office, presence color comes from the collab service and I think this will also be the case for Fluid and Teams.